QEfficient/transformers/modeling_attn_mask_utils.py

```diff
@@ -12,6 +12,43 @@
 from transformers.modeling_attn_mask_utils import AttentionMaskConverter
 
 
+def _update_causal_mask(
```

Comment:
Should we name this better? We're not "updating" the causal_mask here. May be `_create_causal_mask`?

Reply:
yes, makes sense, Updated.